lib/sales_analyst.rb

```diff
@@ -122,8 +124,20 @@ def invoice_status(status)
     ((invoice_count.to_f / invoices.count) * 100).round(2)
   end
 
+  def invoice_paid_in_full?(invoice_id)
+    @transaction_repo.paid_in_full?(invoice_id)
+  end
+
+  def invoice_total(invoice_id)
+    calculate_invoice_total(invoice_id) if invoice_paid_in_full?(invoice_id)
+  end
+
   private
 
+  def calculate_invoice_total(invoice_id)
+    @invoice_item_repo.invoice_total(invoice_id)
```

Comment:
Made this helper method solely to get line 132 under the 80 character limit (thanks hound)